cfgov/v1/management/commands/delete_page.py

```diff
@@ -36,7 +35,7 @@ def handle(self, *args, **options):
             raise CommandError('Must supply a slug or an id')
         try:
             page_to_delete.delete()
-            flush_akamai()
+            page_to_delete.flush()
```

Comment:
Actually, I'm not sure why this was ever here.  We don't need to flush after deleting if we're flushing after unpublishing, since going from unpublished to delete makes no difference for what we're displaying on the live site.